model/src/main/java/com/javahelp/model/user/ClientUserInfo.java

```diff
@@ -8,6 +10,7 @@ public class ClientUserInfo extends UserInfo {
     private String lastName;
     private String address;
     private String phoneNumber;
+    private HashMap<String, Integer> attributeMap;
```

Comment:
Rather than a map of attributes, we probably want these to be fields in the class